run_dir/static/js/controls.js

```diff
@@ -0,0 +1,66 @@
+/*
+File: controls.js
+URL: /static/js/controls.js
+Powers /controls/[List type] - template is run_dir/design/controls.html
+*/
+
+$( document ).ready(function() {
+  document.title = 'something else';
+  init_datatable_controls('negative_controls_table');
+});
+
+// Initialize sorting and searching javascript plugin 
+function init_datatable_controls(table_name) {
+  // Setup - add a text input to each footer cell
+  $('#'+table_name+' tfoot th').each( function () {
+    var title = $('#'+table_name+' thead th').eq( $(this).index() ).text();
+    $(this).html( '<input size=10 type="text" placeholder="'+title+'" />' );
+  } );
+
+  var table = $('#'+table_name).DataTable({
+    "paging":false,
+    "info":false,
+    "order": [[ 1, "asc" ]],
+    "destroy": true,
+    dom: 'Bfrti',
+    buttons: [
+      { extend: 'copy', className: 'btn btn-outline-dark mb-3' },
+      { extend: 'excel', className: 'btn btn-outline-dark mb-3' }
+    ]
+  });
+
+  //Add the bootstrap classes to the search boxes
+  if (!table_name || table_name == "negative_controls_table"){ list_name = "#negative_controls_heading"; }
+  else if(table_name == "positive_controls_table"){ list_name = "#positive_controls_heading"; }
```

Comment:
Could be simplified to
```suggestion
  let list_name = "#negative_controls_heading";
  if(table_name === "positive_controls_table"){ list_name = "#positive_controls_heading"; }
```